authors/apps/articles/views.py

```diff
@@ -0,0 +1,86 @@
+import uuid
+import jwt
+from django.shortcuts import render
+from django.template.defaultfilters import slugify
+from .renderers import ArticleJSONRenderer, ListArticlesJSONRenderer
+from rest_framework import status, exceptions
+from rest_framework.exceptions import PermissionDenied
+from rest_framework.response import Response
+from rest_framework.generics import get_object_or_404
+from rest_framework import generics, status
+from .models import Article
+from authors import settings
+
+
+from rest_framework.permissions import (IsAuthenticated,
+                                        IsAuthenticatedOrReadOnly)
+from .serializers import (
+    CreateArticleAPIViewSerializer
+)
+from authors.settings import RPD
+from ..authentication.models import User
+from .models import Article
+from rest_framework import generics
+
+
+class CreateArticleAPIView(generics.ListCreateAPIView):
+    permission_classes = (IsAuthenticated,)
+    queryset = Article.objects.all()
+    serializer_class = CreateArticleAPIViewSerializer
+
+    def post(self, request):
+        """ Method For Posting Article """
+        article = request.data.get('article', {})
+
+        serializer = self.serializer_class(data=article)
+        serializer.is_valid(raise_exception=True)
+        serializer.save(author=request.user)
+
+        response = serializer.data
+        response.update(article)
+
```

Comment:
This is a post, I see `response.update(article)`, not to say your code is wrong, just walk me through your thought process